gallery_dl/extractor/instagram.py

```diff
@@ -395,6 +394,7 @@ class InstagramImageExtractor(InstagramExtractor):
     def __init__(self, match):
         InstagramExtractor.__init__(self, match)
         self.shortcode = match.group(1)
+        self._find_tags = re.compile(r'#\w+').findall
```

Comment:
You need to add this to the `__init__()` of the base class, so that all Instagram extractors have access to it.

The Travis tests currently have lots of `AttributeError: 'InstagramUserExtractor' object has no attribute '_find_tags'` messages in them. You can run those tests locally with `python test/test_results.py instagram` to check if anything broke before pushing your commits.

Reply:
Uhm, right. I was misleaded by the absence of an `__init__()` in the base class `InstagramExtractor`.

A tentative fix is in https://github.com/mikf/gallery-dl/pull/743/commits/1305200bf4c4de4cb288850d21ce550c28ba8c18

I added `__init__()` to the base class, and also had it call `Extractor.__init__()`, since all the subcategory extractors in their `__init__()` are calling `InstagramExtractor.__init__()` (which in the absence of it would resolve in calling `Extractor.__init__()`, am I right?)



It works for me, but please confirm that this indeed makes sense.

Or maybe it would be better to define `self._find_tags()` as a static method instead?